test/SourceKit/CodeFormat/indent-available-macro.swift

```diff
@@ -0,0 +1,6 @@
+@available(iOS 17, *)
+#Preview("") {
+  UIView()
+}
+
+// RUN: %sourcekitd-test -req=format -line=3 -length=1 %s >%t.response
```

Comment:
If you write the response into `%t.response` you should also check its contents.

Reply:
Without the fix this test crashes, and `indent-crashes.swift` which seems to have a goal similar to this did write into `%t.response` without checking its content I thought it would be fine like this.
What do you think I should do?
- Do not write into `%t.response`?
- Write into `%t.response` and check the content? In that case should I change the test to be badly indented on purpose checking it properly gets into the expected shape?

Thanks

Reply:
@ahoppen Thanks for the help! I just made the modifications you asked.
I also checked that running this test without my SourceKit changes still makes SourceKit crash and the test fail.